rest/src/commonMain/kotlin/builder/component/ComponentContainerBuilder.kt

```diff
@@ -0,0 +1,68 @@
+package dev.kord.rest.builder.component
+
+import dev.kord.common.annotation.KordInternal
+import dev.kord.common.entity.SeparatorSpacingSize
+import kotlin.contracts.InvocationKind
+import kotlin.contracts.contract
+
+/**
+ * A builder for an object which can contain [multiple components][ContainerComponentBuilder].
+ *
+ * @see dev.kord.rest.builder.message.MessageBuilder
+ * @see ContainerBuilder
+ */
+public interface ComponentContainerBuilder {
+    @KordInternal
+    public fun addComponent(component: ContainerComponentBuilder)
+
+    /**
+     * Adds a separator with specified [spacing].
+     */
+    public fun separator(spacing: SeparatorSpacingSize): Unit = separator {
```

Comment:
```suggestion
    public fun separator(
        spacing: SeparatorSpacingSize? = null,
        divider: Boolean? = null,
    ): Unit = separator {
        this.spacing = spacing
        this.divider = divider
    }
```

Reply:
I don't really see the point of that, you can just call the seperator function with the builder yourself

Reply:
I agree, but there is no point to have partial member function. My opinion is "all or nothing" :D